backbone.js

```diff
@@ -1170,7 +1170,6 @@
   // if an existing element is not provided...
   var View = Backbone.View = function(options) {
     this.cid = _.uniqueId('view');
-    options || (options = {});
```

Comment:
this is cool because we're `>= 1.7.0` which made a change to `_.pick` for undefined arguments right?